test/modules/aws-acm-import-integration.ts

```diff
@@ -88,7 +88,7 @@ describe('AwsAcm Import Integration Testing', () => {
   `),
   );
 
-  itDocs(
+  it(
```

Comment:
this is not related to a transaction, why are we excluding it from the documentation?

Reply:
Because there is the same exact query a number of blocks ahead. It just did the same thing so I kept the other one.